server/bin/gold/test-6.2.txt

```diff
@@ -107,9 +106,6 @@ drwxrwxr-x          - tmp
 --- pbench-satellite-local tree state
 +++ pbench log file contents
 ++++ pbench-local/logs
-+++++ pbench-backup-tarballs/pbench-backup-tarballs.log
```

Comment:
Something seems wrong with this test output being removed.

Reply:
Earlier ARCHIVE was not being checked in pbench/server/init.py , they were being checked in each script separately. Now ARCHIVE being checked in pbench/server/init.py, so there are no logs that are being produced in that script due to which .log file is not created and removed from logs. Same log have been created in line-2 of same test due to ARCHIVE being checked in pbench/server/init.py